Add ERC: Compressed RPC Link Format

brunobar79 opened the pull request:
Add new ERC for Compressed RPC Link Format with Method-Specific Shortcuts

brunobar79 committed:
Add ERC: Compressed RPC Link Format with Method-Specific Shortcuts

brunobar79 committed:
Fix linter errors

abcoathup commented on erc-draft-compressed-rpc-link-format.md:
```diff
@@ -0,0 +1,1224 @@
+---
+eip: TBD
```
```suggestion
eip: 8050
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.

brunobar79 committed:
Assign number: ERC 8050

brunobar79 committed:
change param name from u to p

SamWilsn committed:
Rename erc-draft-compressed-rpc-link-format.md to erc-8050.md

SamWilsn commented on erc-8050.md:
```diff
@@ -0,0 +1,1224 @@
+- Go: `google.golang.org/protobuf` ≥ v1.33
+- Rust: `prost` ≥ 0.12
+- Swift: `SwiftProtobuf` ≥ 1.25
+- Kotlin/Java: `protobuf-javalite` ≥ 3.24
```
```suggestion
```

This information will bitrot over time (and ERCs become immutable). It's best to avoid mentioning specific libraries/tools/products.

SamWilsn commented on erc-8050.md:
````diff
@@ -0,0 +1,1224 @@
+}
+```
+
+---
````
```suggestion
```

SamWilsn commented on erc-8050.md:
````diff
@@ -0,0 +1,1224 @@
+}
+```
+
+---
````
```suggestion
```

SamWilsn commented on erc-8050.md:
````diff
@@ -0,0 +1,1224 @@
+capabilities["tip_bps"] = utf8Encode('50') // No quotes for numbers
+```
+
+---
````
```suggestion
```

SamWilsn commented on ERCS/erc-8050.md:
```diff
@@ -0,0 +1,1224 @@
+6. **Reconstruct RPC request**: Build JSON-RPC request per shortcut rules
+7. **Validate**: Ensure reconstructed request is well-formed
+
+---
```
```suggestion
```

SamWilsn commented on ERCS/erc-8050.md:
```diff
@@ -0,0 +1,1224 @@
+
+**Note:** Full webhook payload schemas, data types, validation rules, and behavior are defined in ERC-8026. This ERC only specifies how to encode the capability in the payload format. The capability value is stored as UTF-8 JSON bytes per the [Capabilities](#capabilities) encoding rules.
+
+---
```
```suggestion
```

SamWilsn commented on ERCS/erc-8050.md:
```suggestion
```

SamWilsn reviewed (commented):
There are going to be some `eipw` failures. You'll need to fix those as well.

SamWilsn committed:
Rename erc-8050.md to ERCS/erc-8050.md

brunobar79 committed:
lint

brunobar79 commented:
Thanks for the review @SamWilsn!  I've addressed all the comments but there's still one issue on the HTMLPoofer job complaining about links to 8026 since it doesn't exist (still in draft) - any advice on should I address that?

`internally linking to eip-8026, which does not exist`
